medcat/utils/cdb_utils.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import numpy as np
+
+from copy import deepcopy
+from medcat.cdb import CDB
+
+logger = logging.getLogger(__name__) # separate logger from the package-level one
+
+
+class cdb_utils(object):
```

Comment:
Not everything has to be in a class. This isn't java.
Just make the `merge_cdb` method a module level function.